draft-packages/form/KaizenDraft/Form/TextField/TextField.tsx

```diff
@@ -18,6 +17,9 @@ export type TextFieldProps = {
   id: string
   inputType?: InputType
   required?: boolean
+  /**
```

Comment:
If we leverage prop spreading the HTML attributes, we can reduce this list of props significantly :)

Reply:
I've updated the Input primitive to extent InputHTMLAttributes and then used that in the Text Field. As discussed yesterday I've left in the inputValue and defaultInputValue props to allow for a gentle deprecation of these